src/test/kotlin/racingcar/domain/CarKoTest.kt

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.ints.shouldBeExactly
+import io.mockk.every
+import io.mockk.mockk
+import racingcar.RandomNumberUtils
+
+class CarKoTest : DescribeSpec({
+    describe("자동차 이동 테스트") {
+        val randomNumberGenerator = mockk<RandomNumberUtils>()
+
+        val car = Car("테스트 자동차")
+        beforeEach { car.position = 0 }
+
+        it("입력된 숫자가 4 이상이면 이동한다.") {
+            every { randomNumberGenerator.getRandomNumber() } returns 4
+
+            car.move(randomNumberGenerator.getRandomNumber())
```

Comment:
해당 테스트 케이스에서는 `randomNumberGenerator`을 사용할 필요가 없지 않을까요? 🤔 

```suggestion
            car.move(4)
```